ci-operator/config/stolostron/insights-metrics/stolostron-insights-metrics-release-2.3.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: ubi-minimal
     namespace: ocp
     tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
```

Comment:
This one isn't changing to `stolostron`? Same a few lines below.

Reply:
Checked with @schmidtd. This should get updated automatically once we make the org rename changes to our repo docker file.

Reply:
Thanks for checking.